nextmove/src/main/java/no/difi/meldingsutveksling/nextmove/ConversationResource.java

```diff
@@ -82,23 +85,43 @@
     @CollectionTable(name = "props", joinColumns = @JoinColumn(name = "pids"))
     @XmlElement
     private Map<String, String> customProperties;
-    @JsonIgnore
-    @Transient
-    private Arkivmelding arkivmelding;
 
     ConversationResource() {}
 
-    ConversationResource(String conversationId, String senderId, String receiverId, ServiceIdentifier serviceIdentifier,
-                         LocalDateTime lastUpdate, Map fileRefs, Map customProperties){
+    ConversationResource(String conversationId,
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Constructor has 9 parameters, which is greater than 7 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS00107)